src/global_statistics.h

```diff
@@ -2,18 +2,17 @@
 #define NETDATA_GLOBAL_STATISTICS_H 1
 
 // ----------------------------------------------------------------------------
-// global statistics
-
+/// global statistics
 struct global_statistics {
-    volatile uint16_t connected_clients;
-
-    volatile uint64_t web_requests;
-    volatile uint64_t web_usec;
-    volatile uint64_t web_usec_max;
-    volatile uint64_t bytes_received;
-    volatile uint64_t bytes_sent;
-    volatile uint64_t content_size;
-    volatile uint64_t compressed_content_size;
+    volatile uint16_t connected_clients; ///< number of connected clients
+
+    volatile uint64_t web_requests;            ///< Number of web requests.
+    volatile uint64_t web_usec;                ///< ktsaou: Your help needed
+    volatile uint64_t web_usec_max;            ///< ktsaou: Your help needed
```

Comment:
the max time to serve a request. This is reset to zero every time the chart is updated, so it shows the max time for each iteration.